db/sql_implementation.go

```diff
@@ -163,6 +163,21 @@ func (db *DB) ReadWriteTransactionalContext(ctx context.Context, opts *sql.TxOpt
 }
 
 func (db *DB) AddOrders(orders []*types.OrderWithMetadata) (alreadyStored []common.Hash, added []*types.OrderWithMetadata, removed []*types.OrderWithMetadata, err error) {
```

Comment:
This is backwards compatible since calling it with only V3 orders will behave as expected.